docs/changelog.md

```diff
@@ -2,6 +2,8 @@
 
 ## v1.2.0
 
+* feat(mail): Adds new config to send custom `from` and `subject` for LDAP reset
+password emails.
```

Comment:
Should this not go under current, or a new version or has the version not been tagged yet?

Reply:
Version has not been tagged yet :) And yes... current is actually better. Whoever tags it can add the version. Forgot about that.